src/Storefront/Resources/views/storefront/component/privacy-notice.html.twig

```diff
@@ -21,7 +21,8 @@
                         <label class="custom-control-label no-validation"
                                for="acceptedDataProtection">
                             {{ "general.privacyNotice"|trans({
-                                '%url%': path('frontend.cms.page',{ id: config('core.basicInformation.privacyPage') })
```

Comment:
The %url% needs still to be replaced for backward compatibility

Also add above a deprecation like

```
{# @deprecated tag:v6.5.0 - Translation parameter %url% will be removed, use %privacyUrl% and %tosUrl% instead #}
```

Reply:
changed in eb8aba4